pkg/surveyext/editor.go

```diff
@@ -0,0 +1,222 @@
+package surveyext
+
+// This file extends survey.Editor to add an "Optional" flag
+// and change the default editor to nano on unix os.
```

Comment:
Why this change?

I <3 vim, and presumably survey has some good reason to pick it as the default?

Reply:
I <3 vim too but many people try to avoid it. The Github cli also defaults to nano.